src/pages/Content/index.js

```diff
@@ -26,3 +34,28 @@ function click(selector) {
   const element = document.querySelector(selector) || null;
   if (element) element.click();
 }
+
+// a function that listens to click events and stores them in the recordedEvents array
+function attachGlobalEventListeners() {
+
+  // capture all click events
+  document.body.addEventListener('click', function (e) {
+
+    // find the best selector for click target (id/class/tag/attr)
+    const selector = finder(e.target);
+
+    // store selector in recordedEvents
+    recordedEvents.push({
+      type: 'click',
```

Comment:
@agamm Aren't we expecting the recorded events in format of { action: "<action name>", value: "<value>"} ?

Reply:
No, that is the message passing. This is fine.